tests/unitary/test_from_etherscan.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+import boa
+
+crvusd = "0xf939E0A03FB07F59A73314E73794Be0E57ac1b4E"
+voting_agent = "0xE478de485ad2fe566d49342Cbd03E49ed7DB3356"
+
+
+def test_from_etherscan():
+    boa.env.fork(os.environ["ALCHEMY_MAINNET_ENDPOINT"])
```

Comment:
these tests should be in tests/forked:

https://github.com/vyperlang/titanoboa/blob/ddd3db55a0e4ee24f1386306d70e617197ad41ae/tests/integration/fork/conftest.py

and should be using the 'forked' fixture.